api_server.py

```
@@ -81,12 +81,20 @@ def get_crew_list() -> List[Dict]:
+        if table == "crew_availability":
+            curr = conn.execute(
+                "SELECT end_time FROM crew_availability WHERE crew_id = ? AND start_time <= ? AND end_time > ? LIMIT 1",
+                (entity_id, now, now),
+            ).fetchone()
+        elif table == "appliance_availability":
+            curr = conn.execute(
+                "SELECT end_time FROM appliance_availability WHERE appliance_id = ? AND start_time <= ? AND end_time > ? LIMIT 1",
+                (entity_id, now, now),
+            ).fetchone()
+        else:
+            raise ValueError(f"Invalid table: {table}")
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)